packages/react-native-legal/plugin/src/android/withAndroidLegal.ts

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 
 import { type ConfigPlugin, withAndroidManifest } from 'expo/config-plugins';
 
-import { generateAboutLibrariesNPMOutput, scanDependencies } from '../../../plugin-utils/build/common';
+import { generateAboutLibrariesNPMOutput, scanDependencies } from '../../../../shared/build/index';
```

Comment:
should be imported from `@callstack/react-native-legal-shared` the same like in bare plugin

Reply:
Resolved in 6acc6c1fdbaad46e4f39e03e976772bcca08fc42